### PR TITLE
Help App: Phase 1

### PR DESCRIPTION
**Pull request by zfox23**
[DEV-299](https://highfidelity.atlassian.net/browse/DEV-299)

### CODE REVIEW

File: interface/resources/qml/hifi/simplifiedUI/helpApp/about/HelpAbout.qml [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+            ListView {
+                id: audioInputDevices
+                visible: false
+                property string selectedInputDeviceName
+                Layout.preferredWidth: parent.width
+                Layout.preferredHeight: contentItem.height
+                interactive: false
+                delegate: Item {
+                    Component.onCompleted: {
+                        if (HMD.active && selectedHMD) {
```
Comment (RebeccaStankus) [inline interface/resources/qml/hifi/simplifiedUI/helpApp/about/HelpAbout.qml]: Can these be combined into one statement? (same at line 280)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20939/HighFidelity-Beta-PR16071-42705c2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20939/HighFidelity-Beta-Interface-PR16071-42705c2.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20939/HighFidelity-Beta-PR16071-42705c2.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20939/HighFidelity-Quest-Beta-PR16071-42705c2.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20939/HighFidelity-Beta-PR16071-42705c2.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20939/HighFidelity-Beta-Interface-PR16071-42705c2.exe

**Pull request merged**